src/button/button.scss

```diff
@@ -1,64 +1,66 @@
+@import '../utils/theme/variables.scss';
+
 $horizontal-padding: 16px !default;
 $font-size: 14px !default;
 $height: 36px !default;
-$font-weight: semibold !default;
-$background: lightgray !default;
-$color: black !default;
-$border-color: black !default;
+$font-weight: $semibold !default;
+$background: $gradientGrey !default;
+$color: $darkGreyBlue !default;
+$border-color: $silver !default;
 $border-width: 1px !default;
-$border-radius: 3px !default;
-$font-family: sans-serif !default;
+$border-radius: 4px !default;
+$font-family: "Source Sans Pro", sans-serif !default;
```

Comment:
IMO brc shouldn't set `font-family` as it's inherited by default --> it's one of those CSS properties you set once in your app to `body` or `html`. @veej do you agree?